app/models/post.rb

```diff
@@ -1,4 +1,11 @@
 class Post < ApplicationRecord
```

Comment:
[Required]
You're validating title and body twice. Delete the ones you're not intending to use.